materials/r-intro.md

```diff
@@ -5,47 +5,56 @@ title: Introduction to R
 language: R
 ---
 
-* Programming language
-* Statistics and data analysis environment
+> Remind students to install R and RStudio.
 
+> Have students open RStudio and check to see if console "sees" R.
 
-## Setup
+### R
 
-* Does everyone have R and RStudio installed?
-* Open RStudio, check to see if console sees R
+* Programming language
+* Statistics and data analysis environment
 
 
-## RStudio
+### RStudio
 
 * IDE - Integrated Development Environment
-  * Text editor, with highlighting and information about problems with code
-  * Interpreter
-  * Project management
-
-* Working directly in interpreter
-* Writing code and running it from the editor
-* Clearing the global environment
-
-* Information in Environment/History
-* Integrated help (start typing, F1 for more info)
-
-
-## Projects
+    * Interpreter/Console
+    * Text editor
+        * object highlighting 
+        * information about problems with code
+        * `tab` key autocompletes
+            * Let the computer do repetitious work. 
+            * It's easier and with fewer mistakes.
+    * Environment/History
+    * Project management
+
+> Demo
+>
+> * working directly in interpreter.
+> * writing code and running it from the editor.
+> * clearing the global environment.
+> * Integrated help (start typing, `F1` for more info).
+
+
+### Projects
 
 * Who here uses `setwd()`?
 * Start a new project
 * Create a `data` subdirectory
-* Add an example csv file
+* Add an example `.CSV` file
```

Comment:
Oh, sorry I hadn't noticed this before. Yes, we should definitely change them to lower case since almost every use of them in the real world is lower case and most operating systems and programming languages are case sensitive.